package milkshakeiii;

enum PieceType
{
	NO_PIECE = 0;
	SQUARE = 1;
	UP_TRIANGLE = 2;
	CIRCLE = 3;
}

message GameTurn
{
	optional string game_id = 1;
	optional PieceType placed_piece_type = 2;
	optional int32 column_place = 3;
	optional int32 row_place = 4;
}
